packages/client/hmi-client/src/views/DataExplorer.vue

```diff
@@ -1,13 +1,26 @@
 <template>
 	<div class="data-explorer-container">
 		<div class="facets-and-results-container">
-			<facets-panel
-				v-if="viewType === ViewType.LIST"
-				class="facets-panel"
-				:facets="facets"
-				:filtered-facets="filteredFacets"
-				:result-type="resultType"
-			/>
+			<slider content-width="20%" tab-width="48px" direction="left" :is-open="isSliderFacetsOpen">
+				<template v-slot:content>
+					<div class="slider-header content">
+						<IconChevronLeft24 class="slider-header-item" @click="isSliderFacetsOpen = false" />
+						<h4 class="slider-header-item">Facets</h4>
+					</div>
```

Comment:
This seems something that we might use a lot.
Could you make it a slot `header` like in the modal component?